### PR TITLE
Improve text on the welcome page

### PR DESCRIPTION
**Pull request by regularjack**
The text on the welcome page was not a proper English sentence. I took the chance to also improve the "read the docs" sentence to not mention Symfony twice.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@regularjack thanks for your pull request. Before merging it, I'd like to ask the review from @weaverryan, our native English speaker. Thanks!

File: app/Resources/views/default/index.html.twig [hunk 1]
```diff
@@ -11,7 +11,7 @@
                 <p>
                     <svg id="icon-status" width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1671 566q0 40-28 68l-724 724-136 136q-28 28-68 28t-68-28l-136-136-362-362q-28-28-28-68t28-68l136-136q28-28 68-28t68 28l294 295 656-657q28-28 68-28t68 28l136 136q28 28 28 68z" fill="#759E1A"/></svg>
 
-                    Your application is ready to start working on it at:
+                    You're ready to start working on your application at:
```
Comment (fabpot) [inline app/Resources/views/default/index.html.twig]: I would split this in 2 sentences, something along the lines of:

"Your application is ready now. Start working on it at:"

**Comment by Pierstoval** [comment]
Apart @fabpot's comment, I'm :+1: on this

**Comment by regularjack** [comment]
I also agree with @fabpot's comment. Just fixed it.

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1: sounds better to me too

**Comment by fabpot** [comment]
Thank you @regularjack.

**Pull request closed without merging**